addons/road-generator/nodes/road_segment.gd

```diff
@@ -188,6 +188,50 @@ func check_rebuild() -> bool:
 	return false
 
 
+func generate_edge_curves():
+	if not is_instance_valid(container):
+		return
+	if not is_instance_valid(start_point) or not is_instance_valid(end_point):
+		return
+	if not container.create_edge_curves:
+		clear_edge_curves()
+		return
+
+	# Find the road edge positions
+	var matched_lanes = self._match_lanes()
+	if len(matched_lanes) == 0:
+		return
+
+	var _par = get_parent()
+
+	# Add edge curves
+	var edge_R: Path = _par.get_node_or_null("edge_R")
```

Comment:
My only suggestion out of reviewing the whole PR is that it might be nice to define constants for "edge_R", "edge_F", and "edge_C" as global constants at the top of this file.

You are welcome to do that in the PR for the center "edge" instead of here since it's such a quick change.